modules/libcom/src/osi/os/vxWorks/osdEnv.c

```diff
@@ -86,14 +87,12 @@ LIBCOM_API void epicsStdCall epicsEnvUnset (const char *name)
  */
 LIBCOM_API void epicsStdCall epicsEnvShow (const char *name)
 {
-    if (name == NULL) {
-        envShow (0);
-    }
-    else {
-        const char *cp = getenv (name);
-        if (cp == NULL)
-            printf ("%s is not an environment variable.\n", name);
-        else
-            printf ("%s=%s\n", name, cp);
+    extern char **ppGlobalEnviron;
```

Comment:
Might be worth a comment: `/* Used in 'environ' macro but not declared in envLib.h */`